pam/pam.go

```diff
@@ -35,8 +35,21 @@ const (
 	// PAM module for the second stage authentication to select the default
 	// broker for the current user.
 	authenticationBrokerIDKey = "authentication-broker-id"
+
+	authenticatedKey = "authenticated-key"
```

Comment:
The key is rather `already-authenticated-flag` or something along the line to explain this is to prevent reentry?